app/main/forms/auth_forms.py

```diff
@@ -9,7 +10,9 @@ class LoginForm(Form):
         'Email address', id="input_email_address",
         validators=[
             DataRequired(message="You must provide an email address"),
-            Email(message="You must provide a valid email address")
+            Regexp("^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$",
```

Comment:
I think this regex is definitely going to be too restrictive.  What about extended unicode, which is perfectly valid in an email address, e.g. `ñoñó1234@gov.uk`.

On reflection (and after reading this gem: http://stackoverflow.com/questions/201323/using-a-regular-expression-to-validate-an-email-address) I'm tempted to stick with the WTForms validation out of the box, and just fix the API to not break in the case of multiple `@` signs.

No user account is set up until they click the link in their email, so that's really the proper test that the thing they typed in is a valid email address.  We should just make it so our app doesn't break if someone types the wrong thing in.